### DISCUSSION TITLE
Menu is not working in Linux?

### DISCUSSION THREAD

**Discussion by sigkydj** [NativePHP/laravel#210] [General]
First of all, excellent project, it promises a lot.
I am a web programmer, I am testing NativePHP in test projects but I have noticed that the menu does not work for me, I can put it with HTML, CSS but that is not the case, I use Linux for development, I don't know if because it is a beta version this section is not supported?

**Comment by martinpl** [comment]
For me window menu and menu bars works fine on Arch Linux (btw) / GNOME.

**Comment by simonhamp** [comment]
I'm closing this thread as it sounds like you have an issue. I've created #299 from this thread. @djhyll if you could provide more context there, that would be super helpful.
